integr8ly/integreatly-operator
# MGDAPI-5872 : Create e2e test for ClusterObjectTemplates state

# Issue link

https://issues.redhat.com/browse/MGDAPI-5872

# What

This PR adds an e2e test to ensure that ClusterObjectTemplates are in an active state when RHOAM is fully installed.
As this test depends on the package operator the test will not be running in the prow test suite.

# Verification steps



## Resources

### Indexes
**Feature:** quay.io/jfitzpat/managed-api-service-index:1.39.0-5872

## Verification
* Install RHOAM, the obo `Feature` index can be used
* Wait for installation
* From this branch run
```shell
INSTALLATION_TYPE=managed-api TEST="Verify ClusterObjectTemplates ready state" make test/e2e/single
```
* **Expected:** test should pass successfully.
